tools/serverpod_cli/bin/generator/class_generator_dart.dart

```diff
@@ -12,6 +12,12 @@ String serverpodUrl(bool serverCode) {
       : 'package:serverpod_client/serverpod_client.dart';
 }
 
+String serverpodProtocolUrl(bool serverCode) {
```

Comment:
`serverpodProtocolPackage`? or is Url commonly used by Dart?

Reply:
I think it's a url, right? `schema:server/path`
There is a method called serverpodUrl from my serialization pr. That's mainly why I used this name.

Reply:
I think this is resolved.